app/src/main/res/values-pl/strings.xml

```diff
@@ -0,0 +1,109 @@
+    <string name="cardinal_direction_southeast">Południowy wschód</string>
+    <string name="cardinal_direction_south">Południe</string>
+    <string name="cardinal_direction_southwest">Południowy zachód</string>
+    <string name="cardinal_direction_west">Zachód</string>
+    <string name="cardinal_direction_northwest">Północny zachód</string>
+
+    <string name="cardinal_direction_north_abbreviation">N</string>
+    <string name="cardinal_direction_east_abbreviation">E</string>
+    <string name="cardinal_direction_south_abbreviation">S</string>
+    <string name="cardinal_direction_west_abbreviation">W</string>
+
+    <string name="sensor_accuracy_no_contact">Sensor niedostępny</string>
+    <string name="sensor_accuracy_unreliable">Wartości sensora nie są wiarygodne</string>
+    <string name="sensor_accuracy_low">Niska dokładność</string>
+    <string name="sensor_accuracy_medium">Przeciętna dokładność</string>
+    <string name="sensor_accuracy_high">Maksymalna dokładność</string>
+    <string name="sensor_error_message">Sensor niedostępny</string>
+    <string name="sensor_calibration_text">W dowolnym momencie możesz spróbować poprawić dokładność czujnika, obracając urządzenie w sposób przedstawiający cyfrę 8.</string>
+
+    <string name="location_loading">Oczekiwanie na aktualizację lokalizacji</string>
+    <string name="location_error_message">Lokalizacja niedostępna</string>
+    <string name="location_not_present">
+Nie udało się ustalić lokalizacji. Powrót na północ magnetyczną.</string>
+    <string name="location_reload">Ponownie załaduj lokalizację</string>
+    <string name="access_location_permission_rationale_title">
+Potrzebujemy pozwolenia na dostęp do Twojej lokalizacji</string>
+    <string name="access_location_permission_rationale_message">
+Aby wyświetlić północ geograficzną, aplikacja musi określić deklinację magnetyczną na podstawie Twojej bieżącej lokalizacji.</string>
+    <string name="access_location_permission_denied">Brak pozwolenia na dostęp do lokalizacji. Powrót na północ magnetyczną.</string>
+
+    <string name="settings">Ustawienia</string>
+
+    <string name="true_north">Północ geograficzna</string>
+    <string name="true_north_summary">Skoryguj deklinację magnetyczną i wskaż północ geograficzną. Wymaga dostępu do Twojej lokalizacji.</string>
+    <string name="magnetic_north">Północ magnetyczna</string>
+
+    <string name="haptic_feedback">Feedback dotykowy</string>
+    <string name="haptic_feedback_summary">Kompas wibruje gdy zmienia się kierunek</string>
+
+    <string name="display">Wyświetlanie</string>
+    <string name="night_mode">Motyw</string>
+    <string name="night_mode_follow_system">Ustawienia systemowe</string>
+    <string name="night_mode_no">Jasny</string>
+    <string name="night_mode_yes">Ciemny</string>
+
+    <string name="about">O nas</string>
+    <string name="author">Autor</string>
+    <string name="license">Licencja</string>
```

Comment:
Two new strings were introduced here recently:

    <string name="third_party_licenses">Third-party licenses</string>
    <string name="third_party_licenses_summary">Licenses of included open source libraries</string>

Reply:
No problem at all, I added the missing translations